geonode/base/templates/base/user_and_group_permissions.html

```diff
@@ -0,0 +1,22 @@
+{% extends "geonode_base.html" %}
+{% load i18n %}
+{% load staticfiles %}
+{% load bootstrap_tags %}
+
+{% block title %} {% trans "Batch Edit" %} - {{ block.super }} {% endblock %}
+
+{% block body_class %}{% trans "batch edit" %}{% endblock %}
+
+{% block body %}
+<div class="page-header">
+  <h2 class="page-title">{% trans "Batch Edit" %}</h2>
+</div>
+<form action="/{{ model|lower }}/layer/permission/" method="post">
```

Comment:
is it possible to use {% url ...} here?

Reply:
Yes, I'll use {% url ...%} but it will introduce extra logic because we need to add arguments in {% url ...%} based on the model using the form.